fiatconnect-api.md

```diff
@@ -4,37 +4,37 @@
 
 - [0. Table of Contents](#0-table-of-contents)
 - [1. Introduction](#1-introduction)
-  * [1.1. A Note on Celo & Multi-Chain Support](#11-a-note-on-celo---multi-chain-support)
-  * [1.2. Notational Conventions](#12-notational-conventions)
+  - [1.1. A Note on Celo & Multi-Chain Support](#11-a-note-on-celo---multi-chain-support)
```

Comment:
It looks like some IDE might have performed some automated formatting to the document -- could you please revert those changes?

Reply:
Done